src/aap_eda/core/management/commands/run_rsyslog_configurer.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+
+from aap_eda.utils.external_logging import reconfigure_rsyslog
+
+logger = logging.getLogger("aap.gateway.rsyslog_configurer")
```

Comment:
I guess that since this PR is still WIP you already have in mind to change it, just in case I remind we should not point to gateway stuff in any case.

Reply:
Great catch :) will fix